tests/functional/i/inconsistent/inconsistent_returns_noreturn.py

```diff
@@ -53,3 +53,30 @@ def bug_pylint_4122_bis(s):
         return n
     except ValueError:
         parser_error_name('parser error')
+
+class ClassUnderTest:
+    def _no_return_method(self) -> typing.NoReturn:
+        sys.exit(1)
```

Comment:
```suggestion
    def _no_return_method(self) -> typing.NoReturn:
        sys.exit(1)
        
    def _falsely_no_return_method(self) -> typing.NoReturn:
        return 1
```

Reply:
Resolved by https://github.com/pylint-dev/pylint/pull/8750/commits/ab7679406c63111bcda01bcdeb92b7a08064e33d


See https://github.com/pylint-dev/pylint/pull/8750#discussion_r1221019911 for full context